lib/app.rb

```diff
@@ -2,7 +2,39 @@
 require_relative 'simulator'
 
 class App
```

Comment:
I am thinking of renaming this to `class CLI` which also involve renaming the file to `CLI.rb`.
`App` now feels a bit too general and all the class methods makes more sense in a CLI context.